MyPlayground3 4.25.53 PM.playground/Contents.swift

```diff
@@ -0,0 +1,115 @@
+import UIKit
+
+/*var num: Int?
+num = 65
+print (num)
+
+
+
+var age = "12"
+var ageINT = Int(age)
+print (age)
+
+
+
+var age2Int = "abc"
+var age2 = Int(age2Int)
+print (age2Int ?? 0)
+ */
+
+
+
+// Dictionaies
+/*
+var dic: Dictionary<String,Int>
+dic = ["ali":22,"raghad":33]
+print(dic)
+
+dic.updateValue(25, forKey:"ali")
+print(dic)
+
+ dic["mohammed"] = 50
+ print(dic)
+
+dic.removeValue(forKey: "mohammed")
+//dic["mohammed"]= nill
+ */
+
+
+
+
+
+
+
+/*set
+var mypurchases : Set = ["laptop", "book" , "cookie"]
+var frindpurchases : Set = ["laptop", "book" , "t-shirt" , "watch"]
+
+var myrecommendnextpurchases : Set<String>?
+var frindrecommendnextpurchases : Set<String>?
+
+frindrecommendnextpurchases = frindpurchases.subtracting(mypurchases)
+
+print("recommends \(frindrecommendnextpurchases ?? Set(["nothing"]) ) ")
+
+if let nextPurchase = myrecommendnextpurchases {print(nextPurchase)}
+ */
+
+
+
+
+
+
+
+
+
+
+
+
+
+// Exersise Dic
+
+var Ahmed = [String:Int]()
```

Comment:
Try to use the coding standards while naming variables. in Swift its recommended to start with a small case. In this case its better to name like
**var ahmad**

other example 
**var isValidNumber: Bool**